packages/thumbprint-react/components/StarRating/index.tsx

```diff
@@ -35,111 +24,67 @@ interface PropTypes {
-                onMouseLeave={onMouseLeave}
-                style={{ width: isNumber(ratio) ? `${ratio * 100}%` : undefined }}
-            >
-                {times(MAX_NUM_STARS, index => (
-                    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
-                    <div
-                        key={index}
-                        className={classNames({
-                            [styles.starContainerSmall]: size === 'small',
-                            [styles.starContainerMedium]: size === 'medium',
-                            [styles.starContainerLarge]: size === 'large',
-                        })}
-                        onClick={(): void => onStarClick(index + 1)}
-                        onMouseEnter={(): void => onStarHover(index + 1)}
-                    >
-                        <StarIcon
-                            fill={tokens.tpColorYellow}
-                            className={classNames({
-                                [styles.starSVG]: true,
-                                [styles.starSVGSizeSmall]: size === 'small',
-                                [styles.starSVGSizeMedium]: size === 'medium',
-                                [styles.starSVGSizeLarge]: size === 'large',
-                            })}
-                        />
-                    </div>
-                ))}
-            </div>
-
-            <div className={styles.row} data-test-id="star-row-gray" onMouseLeave={onMouseLeave}>
-                {times(MAX_NUM_STARS, index => (
-                    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
-                    <div
-                        key={index}
-                        className={classNames({
-                            [styles.starContainerSmall]: size === 'small',
-                            [styles.starContainerMedium]: size === 'medium',
-                            [styles.starContainerLarge]: size === 'large',
-                        })}
-                        onClick={(): void => onStarClick(index + 1)}
-                        onMouseEnter={(): void => {
-                            onStarHover(index + 1);
-                        }}
-                    >
-                        <StarIcon
-                            fill={tokens.tpColorGray}
-                            className={classNames({
-                                [styles.starSVG]: true,
-                                [styles.starSVGSizeSmall]: size === 'small',
-                                [styles.starSVGSizeMedium]: size === 'medium',
-                                [styles.starSVGSizeLarge]: size === 'large',
-                            })}
-                        />
-                    </div>
-                ))}
-            </div>
+            {(onStarClick !== noop || onStarHover !== noop) && (
+                <form className={styles.rateForm}>
```

Comment:
We probably shouldn't put this inside of a `form` because then consumers may end up having a `form` within a `form`. Plus, there's no way for them to customize this `form` with a `method` or `action`.

Reply:
Fixed